src/pocapi.cpp

```diff
@@ -35,6 +35,24 @@ using ovms::InferenceTensor;
 using ovms::Status;
 using ovms::StatusCode;
 
+void OVMS_StatusDelete(OVMS_Status* status) {
+    delete reinterpret_cast<ovms::Status*>(status);
+}
+
+OVMS_Status* OVMS_StatusGetCode(OVMS_Status* status,
+    uint32_t* code) {
+    ovms::Status* sts = reinterpret_cast<ovms::Status*>(status);
+    *code = static_cast<uint32_t>(sts->getCode());
+    return nullptr;
```

Comment:
here I wonder if the return of uint32_t wouldn't be better.

Reply:
I should add a check if status is nullptr. If we remove possibility of returning status then we cannot return an error

Reply:
done